.github/workflows/clair-in-ci.db.yaml

```diff
@@ -29,14 +29,14 @@ jobs:
           fi
 
       - name: Log into registry ${{ env.REGISTRY }}
-        uses: docker/login-action@v2.2.0
+        uses: redhat-actions/podman-login@v1
         if: ${{ github.event.client_payload.pr_event != 'pull_request' }}  # don't login from PR; secrets are not passed to PRs from fork
         with:
           registry: ${{ env.REGISTRY }}
           username: ${{ secrets.HACBS_TEST_QUAY_USER }}
           password: ${{ secrets.HACBS_TEST_QUAY_TOKEN }}
 
-      - name: Setup docker for workflow
+      - name: Setup podman for workflow
```

Comment:
How did you installed podman here?

Reply:
okay, I don't see it being explicitly installed even here https://github.com/redhat-appstudio/hacbs-test/blob/main/.github/workflows/clam-db.yaml